bin/build_daq_container

```diff
@@ -12,6 +12,6 @@ bin/setup_remote forch
 
 echo Starting $image_name build...
 
-docker build . -f docker/modules/Dockerfile.daq --build-arg GIT_URL -t daqf/$image_name
+docker build . -f docker/modules/Dockerfile.daq -t daqf/$image_name
```

Comment:
why remove this?

Reply:
I thought that arg wasn't getting consumed anywhere. Is that a mistake?

Reply:
I forget exactly how it works, but it sets an environment variable which is consumed somewhere down the chain of things. Docs seem to imply that it also requires an ENV setting in the Dockerfile, so not sure if it works without that or if it's broken.

Reply:
Fair enough. I reverted that particular bit. Thought I was cleaning up something. Better not to break things.